core/src/main/java/com/linecorp/armeria/client/logging/ContentPreviewingClient.java

```diff
@@ -151,6 +151,7 @@ public HttpResponse execute(ClientRequestContext ctx, HttpRequest req) throws Ex
             final ContentPreviewer requestContentPreviewer =
                     contentPreviewerFactory.requestContentPreviewer(ctx, req.headers());
             req = setUpRequestContentPreviewer(ctx, req, requestContentPreviewer, requestPreviewSanitizer);
+            ctx.updateRequest(req);
```

Comment:
I found we forgot to call `ctx.updateRequest()` thanks to the change I made in [`HttpClientDelegate.java`](https://github.com/line/armeria/pull/4789/files#diff-f0e71f3de27df64a84e1b8fd0ee1b2e505dee7515f66475d77422ea330877d30R67-R72).